src/website/app/demos/Menu/examples/menu-item/customRender.js

```diff
@@ -59,7 +59,7 @@ might be a helpful template. Some things to keep in mind:
           {
             backgroundColor: isHighlighted && theme.color_theme_20,
             display: 'flex',
-            padding: theme.MenuItem_padding,
+            padding: theme.MenuItem_paddingVertical + ' ' + theme.MenuItem_paddingHorizontal,
```

Comment:
IMHO, using a template string here (and in similar scenarios) rather than concatenation would be more readable.  As it stands, one has to read more closely to discern whether math is being done.

```js
padding: theme.MenuItem_paddingVertical + ' ' + theme.MenuItem_paddingHorizontal,
```

```js
padding: `${theme.MenuItem_paddingVertical} ${theme.MenuItem_paddingHorizontal}`,
```

Reply:
I totally agree. Unfortunately, we can’t use string interpolation in our example source, because it is itself already an interpolated string.

Reply:
Oh bummer.  Glad you agree though...